packages/devtools_app/lib/src/screens/app_size/app_size_controller.dart

```diff
@@ -743,7 +743,7 @@ extension AppSizeJsonFileExtension on DevToolsJsonFile {
     'macos',
     'windows',
     'linux',
-    'web'
+    'web',
```

Comment:
this case isn't consistent with how we use trailing commas but seems harmless enough. The best argument in favor for this case is just that the trailing commas make the diffs smaller if you happen to edit the List later.